spec/factories.rb

```diff
@@ -1,2 +1,13 @@
-FactoryBot.define do  
+FactoryBot.define do
+  factory :family do
+    first_name "Ro"
+    last_name "Liso"
+    address "San Cesareo"
+    animals "No"
+    capability 3
+    prof false
+    preferred_gender "female"
+    bus_driver false
+    active true
+  end
```

Comment:
Would be interesting to see if sequencing would work with all these variables,
e.g. sequence(:first_name) { |n| "family first_name#{n}" }
Sequencing would allow multiple families to be created as required, but don't know if you need this for future tests.

Reply:
Oh thanks, probably I will need to add more families for testing, in case I will try it!

Reply:
I would only use sequencing for attributes that need to be unique

Reply:
Thanks Elle, it makes sense! I will keep in mind in case I will need to test unique attributes.